frontend/src/components/Card.tsx

```diff
@@ -108,102 +89,69 @@ const Card = ({
-                    topic={topic}
-                    tooltipLabel={`This project is labeled as "${topic}"`}
-                    type="topic"
-                  />
-                ))}
-                {/* Toggling both languages and topics */}
-                {shouldShowToggleButton && (
-                  <button onClick={handleToggle} className="mt-2 text-gray-600 dark:text-gray-300">
-                    {toggleLanguagesAndTopics ? 'Show less' : 'Show more'}
-                  </button>
-                )}
-              </div>
-            )}
-            {/* Render social links if available */}
-            {social && social.length > 0 && (
-              <div id="social" className="mt-2 flex items-center gap-3">
-                {social.map((item) => (
-                  <a
-                    key={`${item.title}-${item.url}`}
-                    href={item.url}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    className="flex items-center gap-2"
-                  >
-                    <FontAwesomeIcon icon={item.icon as FontAwesomeIconProps['icon']} />
-                  </a>
-                ))}
-              </div>
-            )}
+            <ActionButton tooltipLabel={tooltipLabel} url={button.url} onClick={button.onclick}>
+              {button.icon}
+              {button.label}
+            </ActionButton>
           </div>
-          {/* Action button */}
+        ) : (
           <div
             className={cn(
-              'flex items-center',
-              isMobile && toggleLanguagesAndTopics && 'mt-4 w-full justify-end'
+              'flex w-full items-center justify-between gap-6',
+              isMobile && 'flex-col items-start'
             )}
           >
-            <ActionButton tooltipLabel={tooltipLabel} url={button.url} onClick={button.onclick}>
-              {button.icon}
-              {button.label}
-            </ActionButton>
+            <div className={cn('flex flex-wrap items-center', isMobile && 'w-full')}>
+              {/* Render social links if available */}
+              {social && social.length > 0 && (
+                <div id="social" className="mt-2 flex items-center gap-3">
+                  {social.map((item) => (
+                    <a
+                      key={`${item.title}-${item.url}`}
+                      href={item.url}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="flex items-center gap-2"
+                    >
+                      <FontAwesomeIcon icon={item.icon as FontAwesomeIconProps['icon']} />
+                    </a>
+                  ))}
+                </div>
+              )}
+            </div>
+            {/* Action button */}
+            <div className={cn('flex items-center', isMobile && 'mt-4 w-full justify-end')}>
+              <ActionButton tooltipLabel={tooltipLabel} url={button.url} onClick={button.onclick}>
```

Comment:
There's inconsistent padding know between cards on the right side. It should be the same width as the bottom padding.
![Screenshot 2025-01-17 at 5 39 20 PM](https://github.com/user-attachments/assets/e8f36ab0-7e12-4be2-8908-835dccffee7a)
Whats interesting, I noticed on chapter cards that have social links - the padding looks fine. 
![Screenshot 2025-01-17 at 5 40 22 PM](https://github.com/user-attachments/assets/0a4f494e-cddf-4276-90e2-f73815572ef2)

Can you update this to be consistent on all pages/cards?